README.md

```diff
@@ -28,41 +35,48 @@ Get started writing your application by modifying these key files:
 
 Associated build configuration files and tests are discussed along with the [project structure](#project-structure) below.
 
-### Build
-
-Running the following will build the [zkVM guest program].
+## Test Your Project
+With this Foundry template, you can write a zkVM Rust program in the `/methods/guest` directory which Solidity contracts in  `/contracts` can call into and in return receive a proof or 'receipt' of execution.
+### Test your zkVM program
+To check if your zkVM program will compile, and generate any errors if not, run the Rust compiler with
 
 ```bash
 cargo build
 ```
 
-While the following will build your Ethereum contracts.
-
+If you've written tests in your zkVM progam, run them with
```

Comment:
As a future improvement, it might be nice to include a sample test so that `cargo test` works out of the box.

Reply:
I guess it already works out of the box and just says 0 tests passed. /shrug